autobump-github/src/test/java/com/github/autobump/github/model/GithubReleaseNotesSourceTest.java

```diff
@@ -84,12 +83,12 @@ void getReleaseNotes_ThrowsGithubUnauthorizedException() {
                 .isThrownBy(() -> githubReleaseNotesSource.getReleaseNotes(TEST_URL, TEST));
     }
 
-    @Test
+    /*@Test
     void getReleaseNotes_ThrowsGithubNotFoundException() {
         setupErrorStub(404);
         assertThatExceptionOfType(GithubNotFoundException.class)
                 .isThrownBy(() -> githubReleaseNotesSource.getReleaseNotes(TEST_URL, TEST));
-    }
+    }*/
```

Comment:
this test method should assert that the `githubReleaseNotesSource.getReleaseNotes(TEST_URL, TEST)` method returns `null`

Reply:
Done. Have changed the methodname to getReleaseNotes_WhenGithubNotFoundReturnsNull()